test/cmake/find_modules/CMakeLists.txt

```diff
@@ -3,16 +3,19 @@ project(find_modules)
 
 add_executable(test_prog test.c)
 
+find_package(OpenGL REQUIRED)
 find_package(OpenGL REQUIRED)
```

Comment:
Why do we need these duplicate lines here and bwlow?

Reply:
Just to test that its safe to find the package more than once.

Reply:
Oh, maybe add a comment then. Otherwise the test code looks like it might be a copy-paste error I think.